internal/cmd/kafka/command_cluster_describe.go

```diff
@@ -75,11 +74,7 @@ func (c *clusterCommand) describe(cmd *cobra.Command, args []string) error {
 
 	cluster, httpResp, err := c.V2Client.DescribeKafkaCluster(lkc, environmentId)
 	if err != nil {
-		environmentId, err = pconv.ConvertEnvironmentNameToId(environmentId, c.V2Client)
-		if err != nil {
-			return err
-		}
-		lkc, err = pconv.ConvertClusterNameToId(lkc, environmentId, c.V2Client)
+		lkc, environmentId, err = c.clusterAndEnvNamesToIds(lkc, environmentId)
```

Comment:
If we're potentially passing in a name, why is this variable called "lkc"? That's in reference to an ID.